install.md

```diff
@@ -227,6 +227,43 @@ To configure the installation settings:
      toNamespace: cf
    EOF
    ```
+ 
+If using the experimental Choreographer integration, this SecretExport will also need to grant access to the secret in every cf space namespace. To do so, either:
```

Comment:
Choreographer should be Cartographer no?

Reply:
We appear to use both names, but Choreographer is used more consistently when talking about the integration.